test/functional/fixtures/ui/testcafe-fixtures/selector-inspector-test.js

```diff
@@ -283,3 +283,33 @@ test('should hide panel', async t => {
 
     await t.debug();
 });
+
+test('should indicate the correct number of elements matching the TestCafe selector passed in debug', async t => {
```

Comment:
Your tests aren't executed. You should add them [here](https://github.com/DevExpress/testcafe/blob/d55e8b8145a1d32c9665893817295f6fa70e69ad/test/functional/fixtures/ui/test.js#L45)